R/init-package.R

```diff
@@ -32,7 +32,11 @@
   apiConfig$dimensionFilePath <- filePathFor("OSPSuite.Dimensions.xml")
   apiConfig$pkParametersFilePath <- filePathFor("OSPSuite.PKParameters.xml")
 
-  rSharp::callStatic("OSPSuite.R.Api", "InitializeOnce", apiConfig)
```

Comment:
According to @rwmcintosh we don't have to call `InitializeOnce`  of `OSPSuite.R.Api` because it is done in `rSharp::callStatic("MoBi.R.Api", "InitializeOnce", apiConfig)`